src/app/(app)/(library)/library/[productId]/page.tsx

```
@@ -0,0 +1,35 @@
+      productId,
+    })
+  );
+  void queryClient.prefetchQuery(
+    trpc.reviews.getOne.queryOptions({
+      productId,
+    })
+  );
+  return;
+  <HydrationBoundary state={dehydrate(queryClient)}>
+    <ProductView productId={productId} />
+  </HydrationBoundary>;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)